lib/payload_generator.py

```diff
@@ -51,7 +51,24 @@ def generate_users_payload_multiple_array(self, user_amount):
         return required_payload
 
 
-class UserCredentials:
+class UserCredentials(UserPayload):
+    fake = Faker()
+
     username = fake.simple_profile().get("username")
     password = fake.password
 
+
+class StorePayload:
+    def __init__(self, pet):
+        self.pet = pet
+
+    def generate_place_pet_store_payload(self, true=None):
+        required_payload = {
+                "id": random.randint(1, 10),
+                "petId": self.pet.add_pet_and_get_id(),
+                "quantity": random.randint(1, 5),
+                "shipDate": "2024-09-10T11:57:49.135Z", # hardcoded date?
```

Comment:
One more point to improve:

Please, generate the following date format using `datetime` library:
https://docs.python.org/3/library/datetime.html

Reply:
done